services/forgetPasswordService.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by shujaatali on 04/05/16.
+ */
+var config = require('../nodejs_config/config.js');
+var jwt = require('jsonwebtoken');
+var mailgun = require('mailgun-js')({apiKey: config.App.mailGunApiKey, domain: config.App.mailGunDomain});
+var db = require('nano-blue')(config.App.CouchServerIp+'/members');
+exports.checkUserName = function(req,res){
+
+
+        db.view('cbs', 'getMemberByUsername', {key: req.body.username, include_docs: true}, function(err, respBody) {
+        if (err) {
+        return err;
+             } else {
+            if (respBody.rows.length === 0) { // a doc with this 'username' does not exist in db
+            return 0;
+        } else {
+            var member = respBody.rows[0].doc;
+            var token = jwt.sign({ id: respBody.rows[0].doc._id }, config.App.secretKey,{
+                expiresIn : 259200
+            });
+            console.log(token);
+            var resetPassLink = config.App.serverIp+'/#/resetPassword/'+token;
+            var data = {
+                from: 'Mailgun Sandbox <postmaster@sandboxe91368ca64a2490d9d308fd1d4df5971.mailgun.org>',
+                to: 'm.ali.hussan.hussain@gmail.com',
```

Comment:
yar we have to do something so that email goes to the person who is resetting his password, not to a fixed address.